homeassistant/components/abode/camera.py

```diff
@@ -82,8 +82,21 @@ def camera_image(self):
 
         return None
 
+    def turn_on(self):
+        """Turn on camera."""
+        self._device.privacy_mode(True)
+
+    def turn_off(self):
+        """Turn off camera."""
+        self._device.privacy_mode(False)
```

Comment:
If privacy mode disables the stream, then turning a camera off should enable privacy mode?

Reply:
That is actually correct because I’m an idiot and got it backwards in [abodepy](https://github.com/MisterWil/abodepy/blob/eb9fbbb4b7305e2186595bd764e8a9dfa9eba606/abodepy/devices/camera.py#L121).

I can change it the next time we need to make an abodepy update and made a PR to swap this as well.

Reply:
Since we already missed the beta of 113, we have 3,5 weeks until 114. Should we wait for a new release of abodepy so we don't forget to swap this once we upgrade ?

Reply:
Yeah that sounds good. I just made [PR](https://github.com/MisterWil/abodepy/pull/72) in abodepy. Once it's merged and a new version is pushed, I'll update this one.